### DISCUSSION TITLE
Why not use astro?

### DISCUSSION THREAD

**Discussion by Jak2k** [thedevdojo/static#8] [Q&A]
In my experience, developing with astro is really fun and it has way more features than *static*. Why should I use *static*?

**Comment by giampieropace** [comment]
Personally I find static more intuitive and friendly than astro, eg: I'm more comfortable looping elemets using <foreach> than map(),  or using includes than importing components.

Even using files with .html extension instead of .astro (as well as .jsx .svelte, etc) feels more like "home" to a person like me, an "old internet" nostalgic but who likes modern html centric solutions like tailwind, alpine and htmx.

**Reply by Jak2k** [reply]
I think we should rate a framework by its features and not by its file extensions.The *static* project has not very much features, so I use astro.

**Comment by tnylea** [comment]
I've been working on a V2 of Static that is going to include Tailwind V4 and a bunch of other goodies.

@giampieropace 🙏 Thanks for the kind words. Would you mind if I used your comment as a testimonial on the new marketing site?

**Reply by giampieropace** [reply]
Sure, no problem for me. I'll wait to see what V2 brings.
